scripts/dev/format.sh

```diff
@@ -37,9 +37,9 @@ files=$(find \
     -type f \( -name "*.[ch]" -o -name "*.h.in" \) -print)
 
 clang_format_help=$(clang-format --help)
-echo "$clang_format_help" | grep --quiet -- "--Werror"
+echo "$clang_format_help" | grep -- "--Werror"
 clang_format_has_werror=$?
-echo "$clang_format_help" | grep --quiet -- "--dry-run"
+echo "$clang_format_help" | grep -- "--dry-run"
```

Comment:
You have a set -e in that script I take it?

Reply:
Yes. cmd manipulates +/- e when it actually calls the command.